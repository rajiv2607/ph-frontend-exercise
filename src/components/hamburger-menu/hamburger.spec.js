const { HamburgerMenu } = require("..")
import renderer from 'react-test-renderer';


describe('Snap shot test', () => {
    test('testing rendering for Ham component', () => {
        const component = renderer.create(
            <HamburgerMenu/>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})
