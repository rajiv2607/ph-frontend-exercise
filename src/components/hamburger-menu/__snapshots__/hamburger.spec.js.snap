// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snap shot test test1 1`] = `
<svg
  className="hamburger_icon"
  data-testid="hamburger_icon"
  fill="currentColor"
  height="1em"
  onClick={[Function]}
  stroke="currentColor"
  strokeWidth="0"
  style={
    Object {
      "color": undefined,
    }
  }
  viewBox="0 0 512 512"
  width="1em"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M32 96v64h448V96H32zm0 128v64h448v-64H32zm0 128v64h448v-64H32z"
  />
</svg>
`;
